### ISSUE TITLE
bounding boxes

### ISSUE THREAD

**Issue by MatthewNoyce** [RaivoKoot/Video-Dataset-Loading-Pytorch#21]
Hi there, this looks like a really good dataloader, but i was wondering why the bounding boxes werent loaded in the annotations, i have read through the read me briefly and it doesnt look like any bounding boxes are to be included within the annotations files. if this isnt the case please let me know so i can avoid doing extra work thank you.

**Comment by RaivoKoot** [comment]
Hi there. My current implementation does not work with annotation files including anything else but simple class labels. You need to edit the code to also support loading bounding boxes. Good luck :)
